src/qml/main.qml

```diff
@@ -80,6 +101,19 @@ CutieWindow {
                 elide: Text.ElideRight
                 horizontalAlignment: Text.AlignHCenter
             }
+            
+            CutieStore {
```

Comment:
Do not initialize this inside the delegate. That would cause it to initialize identical objects for each button which causes a heavy performance impact.